service/frontend/workflowHandler.go

```diff
@@ -59,13 +60,32 @@ const (
 	defaultLastMessageID                 = -1
 )
 
-var _ workflowserviceserver.Interface = (*WorkflowHandler)(nil)
+const (
+	// HealthStatusOK is used when this node is healthy and rpc requests are allowed
+	HealthStatusOK HealthStatus = iota
```

Comment:
nit: relying on the default value to be 0 and not explicitly set the healthState to `HealthStatusOK` in `NewWorkflowHandler()` is a little bit confusing. Probably follow uber's best practice for enum and use  `HealthStatusOK HealthStatus = iota + 1`

Reply:
that's a good idea. Will do that.